lib/twine/formatters/abstract.rb

```diff
@@ -3,6 +3,8 @@
 module Twine
   module Formatters
     class Abstract
+      LANGUAGE_CODE_WIHT_OPTIONAL_REGION_CODE = "[a-z]{2}(?:-[A-Za-z]{2})?"
```

Comment:
typo! `WIHT`

Reply:
This might be causing an issue for us with `zh-Hant` and `zh-Hans`. With the latest commit, we are not able to consume those android localization files